hydra-haskell/src/main/haskell/Hydra/Langs/Haskell/Serde.hs

```diff
@@ -138,9 +138,9 @@ instance ToTree H.Import where
 instance ToTree H.Literal where
   toTree lit = cst $ case lit of
     H.LiteralChar c -> show $ C.chr $ fromIntegral c
-    H.LiteralDouble d -> show d
-    H.LiteralFloat f -> show f
-    H.LiteralInt i -> show i
+    H.LiteralDouble d -> if d < 0 then "(0" ++ show d ++ ")" else show d
```

Comment:
Let's say `D = |d|`. Why do we need to output it as `(0-D)` instead of just `(-D)`?

Reply:
Haskell is quirky that way. `-` is a binary operator, and only a binary operator, so you have to say `d1-d2`; you can't just say `-d2`. An alternative is `negate d2`.